Jellyfin.Server/Migrations/MigrationRunner.cs

```diff
@@ -75,7 +75,10 @@ public static void RunPreStartup(ServerApplicationPaths appPaths, ILoggerFactory
 
             var xmlSerializer = new MyXmlSerializer();
             var migrationConfigPath = Path.Join(appPaths.ConfigurationDirectoryPath, MigrationsListStore.StoreKey.ToLowerInvariant() + ".xml");
-            var migrationOptions = (MigrationOptions)xmlSerializer.DeserializeFromFile(typeof(MigrationOptions), migrationConfigPath)!;
+            var migrationOptions = File.Exists(migrationConfigPath)
+                 ? (MigrationOptions)xmlSerializer.DeserializeFromFile(typeof(MigrationOptions), migrationConfigPath)!
+                 : new MigrationOptions();
+
```

Comment:
My bad, one line too many
```suggestion
```

Reply:
Ok, I force-pushed to remove the last line without creating another commit.